### DISCUSSION TITLE
Windows network Discovery

### DISCUSSION THREAD

**Discussion by bnilsson11** [rern/rAudio#680] [Feature Requests]
Hi,

The smb file sharing device is not showing up in Windows Network when enabled.
Should it?
Now we have to use "map network drive"  and connect using ip and share.
This is fine, but less convenient.

/BN

**Comment by rern** [comment]
_Edited: Add missing `\\` prefix_ (

~~From: Features > Others > Question icon:~~
...
• At address bar of Windows File Explorer: `\\192.168.1.x` or `\\rAudio`

Web Service Discovery had not been consistent with rAudio. The method above should be convenient enough.

**Reply by rern** [reply]
Sorry.
Wrong info was corrected.
